File: PRAKTIKUM 5/PRAK505_2310817220028_SheilaSabina.py
def biodata(tahun_lahir, nama, asal):
    tahun_sekarang = 2020
    umur = tahun_sekarang - tahun_lahir

    print(f"Perkenalkan Nama Saya: {nama}")
    print(f"Umur Saya: {umur}")
    print(f"Saya Adalah Angkatan: {tahun_sekarang}")
    print(f"Asal Saya dari: {asal}")

def main():
    tahun_lahir = int(input("Input\n"))
    nama = input()
    asal = input()

    print("\nOutput")
    biodata(tahun_lahir, nama, asal)

if __name__ == "__main__":
    main()

File: PRAKTIKUM 5/PRAK505_2310817220028_SheilaSabina.c
#include <stdio.h>

void Biodata(int tahunLahir, char nama[], char asal[]) {
    int tahun_sekarang = 2023;
    int umur = tahun_sekarang - tahunLahir;

    printf("Perkenalkan Nama Saya : %s\n", nama);
    printf("Umur Saya : %d\n", umur);
    printf("Saya Adalah Angkatan : %d\n", tahun_sekarang);
    printf("Asal Saya dari : %s\n", asal);
}

int main() {
    int tahunLahir;
    char Namaku[20], Asal[15];

    printf("Input\n");
    scanf("%d", &tahunLahir);
    scanf(" %[^\n]%*c", Namaku);
    scanf(" %[^\n]%*c", Asal);

    printf("\nOutput\n");
    Biodata(tahunLahir, Namaku, Asal);

    return 0;
}